### PR TITLE
update successful email copy for award winners

### PR DESCRIPTION
**Pull request by macool**
https://trello.com/c/enjh8HLa/635-qae-support-change-successful-business-winners-email-copy-currently-titled-in-settings-as-winners-email-notifying-all-winning-ap

### CODE REVIEW

**Comment by macool** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;enjh8HLa&#x2F;635-qae-support-change-successful-business-winners-email-copy-currently-titled-in-settings-as-winners-email-notifying-all-winning-ap"> QAE Support: Change SUCCESSFUL Business Winners email copy currently titled in &#39;Settings&#39; as &#39;WINNERS : Email notifying all winning applicants of their success, including request for reception attendee details&#39;.</a>

File: app/renderers/mail_renderer.rb [hunk 1]
```diff
@@ -78,7 +78,7 @@ def winners_notification
     assigns[:token] = "secret"
     assigns[:form_answer] = form_answer
     assigns[:name] = "Jon Snow"
-    assigns[:deadline] = deadline("buckingham_palace_attendees_details")
+    assigns[:deadline] = "21 April 2016"
```
Comment (matthewford) [inline app/renderers/mail_renderer.rb]: This shouldn't be hard coded - will ensure the admins set the correct time

File: app/renderers/mail_renderer.rb [hunk 2]
```diff
@@ -105,9 +105,13 @@ def dummy_user(first_name, last_name, company_name)
   end
 
   def form_answer
-    f = FormAnswer.new(id: 0, award_type: "innovation").decorate
-    f.award_year = AwardYear.current
-    f
+    FormAnswer.new(
+      id: 0,
+      urn: "QA0128/16I",
```
Comment (matthewford) [inline app/renderers/mail_renderer.rb]: Not sure about this change

File: app/views/users/buckingham_palace_invite_mailer/invite.html.slim [hunk 3]
```diff
@@ -1,27 +1,52 @@
...
+  ' Congratulations once again on your Award.
 
-p style="font-weight: 700; font-size: 19px; line-height: 1.315789474; margin: 38px 0 38px 0;"
-  ' Thank you,
+p style="font-size: 19px; line-height: 1.315789474; margin: 38px 0 38px 0;"
+  ' Yours sincerely
   br
   br
-  ' The Queen's Awards Office
+  ' The Secretary, Queen’s Awards for Enterprise
```
Comment (matthewford) [inline app/views/users/buckingham_palace_invite_mailer/invite.html.slim]: edit the `’` to be a `'`

Comment (matthewford) [inline app/views/users/buckingham_palace_invite_mailer/invite.html.slim]: in the whole email

Comment (matthewford) [inline app/views/users/buckingham_palace_invite_mailer/invite.html.slim]: and in `app/views/users/buckingham_palace_invite_mailer/invite.text.slim`

File: app/renderers/mail_renderer.rb [hunk 4]
```diff
@@ -105,9 +105,13 @@ def dummy_user(first_name, last_name, company_name)
...
+      award_type: "innovation",
+      award_year: AwardYear.current,
+      award_type_full_name: "Innovation"
```
Comment (matthewford) [inline app/renderers/mail_renderer.rb]: Innovation shouldn't be hard coded

Comment (macool) [inline app/renderers/mail_renderer.rb]: @matthewford `MailRenderer` is used only to preview examples in the UI https://github.com/bitzesty/qae/search?utf8=%E2%9C%93&q=Mailrenderer

Comment (matthewford) [inline app/renderers/mail_renderer.rb]: Alrighty then

**Comment by macool** [comment]
this PR introduces new deadline settings and these need to be created upon deployment

**Pull request merged**